package.json

```diff
@@ -2,7 +2,7 @@
   "name": "angular",
   "license": "MIT",
   "branchVersion": "^1.7.0",
```

Comment:
When should this be updated?

Reply:
https://github.com/angular/angular.js/blob/master/RELEASE.md#dont-update-the-packagejson-branchversion-until-the-cdn-has-updated